specification/app/resource-manager/Microsoft.App/preview/2023-04-01-preview/Jobs.json

```diff
@@ -706,6 +709,21 @@
             }
           }
         },
+        "eventTriggerConfig": {
+          "type": "object",
+          "description": "Manual trigger configuration for a single execution job. Properties replicaCompletionCount and parallelism would be set to 1 by default",
```

Comment:
Event trigger configuration

Reply:
no, we need to match rp and other trigger configs
<img width="187" alt="image" src="https://user-images.githubusercontent.com/41846928/234468129-2b44f993-832b-4f46-90e5-f6bdc96af873.png">

Reply:
@tdaroly check the description, it mentions "Manual trigger configuration" instead of "Event trigger configuration". I know that the property name needs to match RP.